UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/BloodTypes/BloodType.cs

```diff
@@ -69,6 +69,10 @@ public float GetSpareGasCapacity(ReagentMix reagentMix, Reagent reagent = null)
 
 		public float CalculatePercentageBloodPresent(ReagentMix reagentMix)
 		{
+			if (reagentMix.Total == 0)
```

Comment:
```suggestion
			if (reagentMix.Total.Approx(0))
```
It comes from SweetExtensions